device/CLEARFOG.conf

```diff
@@ -0,0 +1,23 @@
+# https://www.solid-run.com/marvell-armada-family/clearfog/
+
+export MAKE_ARGS_DEV="
+CROSS_BINUTILS_PREFIX=/usr/local/arm-none-eabi/bin
```

Comment:
this should use `CROSS_BINUTILS_PREFIX=/usr/local/arm-gnueabi-freebsd${PRODUCT_ABI}/bin`

because arm-none-eabi will cause many problem when compiling kernel, especially with cloudabi and [armelf_fbsd](https://github.com/opnsense/src/blob/master/sys/conf/kern.mk#L280)
if you found any workaround instead of manually editing kern.mk to make it useable with none-eabi, it would be great.

this applies to all armv7h